sdk/src/main/scala/com/horizen/api/http/client/SecureEnclaveApiClient.scala

```diff
@@ -19,70 +18,97 @@ import scorex.util.ScorexLogging
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class SecureEnclaveApiClient(settings: RemoteKeysManagerSettings)(implicit ec: ExecutionContext) extends Actor with ScorexLogging {
+class SecureEnclaveApiClient(settings: RemoteKeysManagerSettings)(implicit system: ActorSystem, ec: ExecutionContext) extends ScorexLogging {
 
-  implicit val system: ActorSystem = context.system
   private[client] val http: HttpExt = Http(system)
   private val keySerializer: SchnorrPropositionSerializer = SchnorrPropositionSerializer.getSerializer
   private val signatureSerializer: SchnorrSignatureSerializer = SchnorrSignatureSerializer.getSerializer
+  val SCHNORR = "schnorr"
 
-  private def serializePublicKey(publicKey: SchnorrProposition) =
-    BytesUtils.toHexString(keySerializer.toBytes(publicKey))
+  def isEnabled: Boolean = settings.enabled
+
+  def listPublicKeys(): Future[Seq[SchnorrProposition]] = {
+    http.singleRequest(
+      Post(settings.address + "/api/v1/listKeys")
```

Comment:
"/api/v1/" is it a requirement from Infra?
if not, you should expect to have "/api/v1/" as a part of `settings.address`

Reply:
yes, that's a requirement from here https://horizenlabs.atlassian.net/browse/INFRA-711